### ISSUE TITLE
Input not supported for this function

### ISSUE THREAD

**Issue by MboiTui** [thierrygosselin/radiator#73]
I encountered this issue trying to run individual filtering functions.  The error traceback was as follows:

Error: Input not supported for this function: read function documentation
4. stop(fallback)
3. signal_abort(cnd)
2. rlang::abort("Input not supported for this function: read function documentation")
1. filter_individuals(gds, interactive.filter=T)

This happened for:
- filter_genotyping
- filter_coverage
- filter_individuals

**Comment by MboiTui** [comment]
I might have found the issue, and it might have been on my side and not a bug.

I was reading the dart file with 'tidy.dart=FALSE', which returned a 'tbl_df' format when running it through 'detect_genomic_format'. Then looking at the function I saw this:

 data.type <- radiator::detect_genomic_format(data)
    if (!data.type %in% c("SeqVarGDSClass", "gds.file")) {
      rlang::abort("Input not supported for this function: read function documentation")

I then changed 'tidy.dart' to FALSE, which returns a SeqVarGDSClass object, and they are now working.

**Comment by MboiTui** [comment]
I might have found the issue, and it might have been on my side and not a bug.

I was reading the dart file with 'tidy.dart=FALSE', which returned a 'tbl_df' format when running it through 'detect_genomic_format'. Then looking at the function I saw this:

data.type <- radiator::detect_genomic_format(data)
if (!data.type %in% c("SeqVarGDSClass", "gds.file")) {
rlang::abort("Input not supported for this function: read function documentation")

I then changed 'tidy.dart' to FALSE, which returns a SeqVarGDSClass object, and they are now working.
